src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+
+interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  onViewMoreProjects: () => void;
+}
+
+const ProjectCardModal: React.FC<PropTypes> = ({ open, onClose, onViewMoreProjects }) => {
+  const [answers, setAnswers] = useState<any[]>([
+    { id: 1, answerOne: "Sample answer one", answerTwo: "Sample answer two" }
+  ]);
+  const [answerOne, setAnswerOne] = useState<string>('');
```

Comment:
Combine answerOne and answerTwo would be better. Really should consider a library for handling forms